lib/l10n.js

```diff
@@ -149,10 +149,12 @@ export function message(profileCode, rule, key, extra) {
                         `[${profile}][${rule.section}][${rule.rule}].text`
                     );
                 }
+                // TODO: to change after 2023
```

Comment:
Please, create an issue listing everything that needs to be rolled back in January so we don't miss these TODOs

Reply:
resolved by https://github.com/w3c/specberus/issues/1686